### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by alexjoel42**
taking this out of Slack

# Overview

The original communication is kind-of untrue. You can apply LPC to RTP CSV protocols. 

## Test Plan and Hands on Testing

N/A

## Changelog

Just changing the release notes slightly

## Review requests

@ecormany confirming that the language is worth changing. 

## Risk assessment

It's a bit wordy and still this concept is going to be pretty tricky to communicate

>Previously saved labware offsets can't be applied to protocols that require selecting a CSV file as a runtime parameter value. Run Labware Position Check freshly every time, unless you are rerunning something from the touch screen with already saved labware offsets and unchanged parameters.
>

Easier to maybe be a little bit less precise and more concise.

### CODE REVIEW

**Review by ecormany — COMMENTED** [review]
I don't think this wording change is meaningful, and it actually hides the two workarounds to the issue that we're proposing:

- Write a clever protocol that obviates the issue. _or_
- Run LPC always.

The issue here is that the **Apply labware offset data** checkbox in the app does nothing or doesn't do what the user expects. 

I would consider adding the word `Stored` at the beginning of the first sentence. But it's also true that you can't LPC any of your labware until the CSV is added and analysis passes.

**Comment by alexjoel42** [comment]
> I would consider adding the word `Stored` at the beginning of the first sentence. But it's also true that you can't LPC any of your labware until the CSV is added and the analysis passes.

I like that comment a lot. While you clarified later in the paragraph, I saw " labware offsets can't be applied to protocols" and promptly lost track of the rest of the paragraph. 

- **Stored** labware offsets can't be applied to protocols that require selecting a CSV file as a runtime parameter value. Write the protocol in such a way that it passes analysis with or without the CSV file, or run Labware Position Check after confirming parameter values.

Would be stronger/truer to me.

**Comment by ecormany** [comment]
Incorporated the small change we settled on as part of #16192.
